lib/utils/httpclient.js

```diff
@@ -33,10 +52,7 @@ module.exports = function(defaultHeaders) {
       logger.debug('Retrieved %s from cache.', url);
     }
 
-    return {
-      contentType: res.headers['content-type'],
-      body: res.body
-    };
+    return new Response(res.headers['content-type'], res.body);
```

Comment:
We should probably replace `got` with a fetch implementation, either bumping the requirement to Node.js 18 to use the stdlib, or use one of the simple wrappers.